libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts

```diff
@@ -41,33 +40,21 @@ export class GroupSyncDetailDialogComponent implements OnInit {
     this.loadGroup();
   }
 
-  onForceStructure(): void {
-    this.loading = true;
-    this.groupService.forceGroupStructureSynchronization(this.data.groupId).subscribe(
-      () => {
-        this.notificator.showSuccess('DIALOGS.GROUP_SYNC_DETAIL.STRUCT_FORCE_SUCCESS');
-        this.loading = false;
-      },
-      () => (this.loading = false)
-    );
-  }
-
   onCancel(): void {
     this.dialogRef.close(null);
   }
 
   onForce(): void {
     this.loading = true;
-    if (this.isBasic()) {
+    if (this.type === 'BASIC') {
```

Comment:
Again please replace the deprecated subscription by the recommended observer argument `{next: (), error: ()}`